src/test/java/org/fundacionjala/coding/german/EANValidatorTest.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.coding.german;
+
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Administrator on 8/29/2017.
+ */
+public class EANValidatorTest {
+
+    /**
+     * Verify validate method.
+     */
+    @Test
+    public void testValidateMethodIsTrue() {
+        EANValidator eanValidatorTest = new EANValidator();
+        boolean result = eanValidatorTest.validate("4003301018398");
+        assertTrue("validate Method should return 'true'", result);
+    }
+
+    /**
+     * Verify validate method.
+     */
+    @Test
+    public void testValidateMethodIsFalse() {
+        EANValidator eanValidatorTest = new EANValidator();
```

Comment:
You are repeating this code in all your @Test 
Please move it to an @Before method.